block/blk-core.c

```diff
@@ -927,6 +929,12 @@ generic_make_request_checks(struct bio *bio)
 		}
 	}
 
+	ret = io_filter_bpf_run(bio);
+	printk(KERN_DEBUG "io_filter_bpf_run returned %d", ret);
+	/*
+	if (io_filter_bpf_run(bio))
+		goto end_io;
```

Comment:
goto end_io makes sense here

Reply:
+1

Reply:
Changed from printing value to using the if statement